### PR TITLE
Admin PM Cancel

### PR DESCRIPTION
**Pull request by Geevies**
* You can now properly cancel out of an Admin PM prompt by pressing cancel.

### CODE REVIEW

**Comment by Geevies** [comment]
!bugfix

**Comment by Geevies** [comment]
!review

**Comment by Geevies** [comment]
okay bls check that latest commit before merg

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -572,13 +572,18 @@ Traitors and the like can also be revived with the previous role mostly intact.
...
+
 	if(!holder)
 		to_chat(src, "Only administrators may use this command.")
 		return
+
 	var/reporttitle
 	var/reportbody
 	var/reporter = null
-	var/reporttype = input(usr, "Choose whether to use a template or custom report.", "Create Command Report") in list("Template", "Custom", "Cancel")
+
```
Comment (NonQueueingMatt) [inline code/modules/admin/verbs/randomverbs.dm:583]:
```suggestion
```

**Pull request merged**